### PR TITLE
2232: Support svgs

### LINKED ISSUE #2232: IGAPP-1305: SVG tiles not shown
### Describe the Bug

SVG tile icons are currently not displayed at all in the app, instead there is just blank space.

### Steps to Reproduce

1. Go to testumgebung > de in MALTE (test cms or normal cms)
2. Look at icon of `Ankommen`, should be missing. In the browser on https://webnext.malteapp.de/testumgebung/de it is displayed correctly.

### Expected Behavior

The icon should be displayed correctly.

### Actual Behavior

Nothing is displayed on native.

### Additional Information

Warning: Icons are sometimes also not displayed in the test cms because they have been deleted.

Make sure:
- offline functionality also works for svg icons
- svgs from the cms are displayed in other places correctly as well

#### Expected Results:
If the icon for category tiles is an svg, it should be shown in the same way then pngs.

#### Actual Results:
SVGs as tile icons are not shown at all.

#### Steps to Reproduce:
* Step_1

Even though its apparently forbidden at the moment to upload svgs, it seems to be still possible some way... Since we will sooner or later have to switch to svgs bc of accessibility issues we might support it now already.

**Environment**: -
**Linked issues:**
- Blocks: IGAPP-1301

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Support SVGs in the native app by using SvgCssUri from react-native-svg.

### Proposed changes

- Use react-native-svgs SvgCssUri to render svgs in SimpleImage component
- Add svgs to resources that are saved offline

### Side effects

None.

### Resolved issues

Fixes #2232.

---

### CODE REVIEW

File: native/src/components/SimpleImage.tsx [hunk 1]
```diff
@@ -48,6 +49,10 @@ const SimpleImage = ({
     return <Image source={source} resizeMode={resizeMode} style={style} />
   }
 
+  if (source.endsWith('.svg')) {
```
Comment (ztefanie) [inline native/src/components/SimpleImage.tsx:52]: `source.endsWith('.svg')` could be moved to a boolean var, so we only need this once.

**Review by ztefanie — COMMENTED** [review]
Code looks good. Not tested. 

How can I test this?

**Comment by steffenkleinle** [comment]
> How can I test this?

See issue description > steps to reproduce :)

**Review by ztefanie — APPROVED** [review]
Looks good on Android emulator. Thanks for this improvement :rocket:

**Review by LeandraH — APPROVED** [review]
Very cool, tested in emulated iOS

**Pull request merged**